test/javascript/components/mentoring/mentor_conversations_list/pagination.test.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import '@testing-library/jest-dom/extend-expect'
+import Pagination from '../../../../../app/javascript/components/mentoring/mentor_conversations_list/pagination.jsx'
```

Comment:
I think it's worth setting up module mappers so we can do `from '~app/` or something like that (not this PR)

Reply:
How do I set that up?

Reply:
That is a good question. I haven't looked at this project setup.

Do the tests run/transpile _through_ webpacker? If yes, it's as easy as using [`additional_paths`](https://github.com/rails/webpacker#resolved). In fact, it might already be set-up. If no, let me know how it is ran.